go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/eapache/channels v1.1.0
 	github.com/eapache/queue v1.0.2 // indirect
 	github.com/fsnotify/fsnotify v1.4.2 // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
```

Comment:
where this is used?

Reply:
Sorry, left over from my debugging, cleaned up in the latest version of PR